client.go

```diff
@@ -87,6 +87,15 @@ func fileInfoFromResponse(resp *internal.Response) (*FileInfo, error) {
 		return nil, err
 	}
 	if resType.Is(internal.CollectionName) {
+		var getMod internal.GetLastModified
+		if err := resp.DecodeProp(&getMod); err != nil && !internal.IsNotFound(err) {
+			return nil, err
+		}
+
+		if !internal.IsNotFound(err) {
```

Comment:
`err` here is not the same variable as the one if the `if` above, because of scoping.